packages/manager/modules/resource-tagging/src/service.js

```diff
@@ -20,4 +26,87 @@ export default class ovhManagerResourceTaggingService {
     if (!defaultKey) return false;
     return !!defaultKey.values.find((val) => val === value);
   }
+
+  assignTag(resourceUrn, key, value) {
+    return this.Apiv2Service.httpApiv2({
+      method: 'post',
+      url: `/engine/api/v2/iam/resource/${resourceUrn}/tag`,
+      data: {
+        key,
+        value,
+      },
+    });
+  }
+
+  getAllUserTags() {
```

Comment:
Can we add some information for readability for this function - getAllUserTags ?